dashboard/test/ui/features/learning_platform/course_versions.feature

```diff
@@ -124,9 +124,3 @@ Scenario: Switch versions using dropdown on script overview page
   And element ".assignment-version-title:contains(2018)" is not visible
   And I click selector ".assignment-version-title:contains(2017)" once I see it
   Then I wait until I am on "http://studio.code.org/s/ui-test-versioned-script-2017"
-
-@as_student
-@no_mobile
-Scenario: Course unit family names redirect to 2019 version
-  When I am on "http://studio.code.org/s/csp3"
-  And I get redirected to "/s/csp3-2019" via "dashboard"
```

Comment:
@bethanyaconnor I think this looks good, based on the fact that the gsheet i linked to at https://github.com/code-dot-org/code-dot-org/pull/41293#issuecomment-869843233 indicates that most of the hits we get to unversioned course unit urls like these are "deep links" to specific levels, not to the unit itself.